scripts/preflight.sh

```diff
@@ -47,7 +47,7 @@ gotesplit \
     -total "$total" \
     -index "$index" \
     github.com/superfly/flyctl/test/preflight/... \
-    -- --tags=integration -v -timeout=60m $test_opts | tee "$test_log"
+    -- --parallel=4 --tags=integration -v -timeout=60m $test_opts | tee "$test_log"
```

Comment:
not sure what is happening with [this build](https://github.com/superfly/flyctl/actions/runs/5660882717/job/15337837456?pr=2607) but all other jobs took less than 5min while that is 10m+ and still running.

I'd lower the test timeout to 10min if we confirm it is stuck due to server side issues.

Reply:
I'm gonna wait for it to timeout, so I can view which specific test failed. I'm pretty sure it's a server side issue though.

Reply:
the test has failed on two consecutive commits, I'm gonna look into it further.

Reply:
i'm thinking this is most likely a backend issue. The [same test](https://github.com/superfly/flyctl/actions/runs/5661445183/job/15339542250) was failing for me on a completely separate commit